js-md5/js-md5-tests.ts

```diff
@@ -1,20 +1,29 @@
+import * as md5 from "js-md5";
```

Comment:
Yes! Didn't catch that, can you switch that to v0.4 if that's the correct version?